### PR TITLE
Add initial project structure;

### PR DESCRIPTION
**Pull request by vlad-dubrovski**
Card: https://vocovo.atlassian.net/jira/software/c/projects/VOC/boards/55?modal=detail&selectedIssue=VOC-3242
Add initial project structure;
Add docker test;

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  devices-manger:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    ports:
+      - 3033:3033
+    environment:
+      NODE_ENV: localhost
+    networks:
+      - devices-manager-network
```
Comment (iryna-markovich-vocovo) [inline docker-compose.yml:13]: should not it be `devices-network` to be common network for devices service and reg module?

File: src/index.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+function main() {
+    console.log('Pong')
+}
+
+main()
```
Comment (iryna-markovich-vocovo) [inline src/index.ts:5]: add please empty lines at the end of each file (where it is required)

Comment (iryna-markovich-vocovo) [inline src/index.ts:5]: add please empty lines at the end of each file (where it is required)

**Review by iryna-markovich-vocovo — CHANGES_REQUESTED** [review]

**Comment by iryna-markovich-vocovo** [comment]
just out of interest do we ok with a name of the repo starts with a capital letter - `Edge-devices-manager`?

File: acceptance-test/docker.test.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import { execSync } from 'child_process'
+import { argv } from 'node:process'
+import { exit } from 'process'
+
+const dockerTag = argv.filter((x) => x.startsWith('--containerTag='))[0]?.split('=')[1]
```
Comment (iryna-markovich-vocovo) [inline acceptance-test/docker.test.ts:5]: dockerTag wrapped in quotes and while exec the docker run comand there is an error of invalid reference format for docker.
maybe this problem is only on windows machines

```suggestion
const dockerTag = argv.filter((x) => x.startsWith('--containerTag='))[0]?.split('=')[1].replace(/['"]+/g, '')
```

Comment (vlad-dubrovski) [reply acceptance-test/docker.test.ts:5]: @iryna-markovich-vocovo yep, on MAC is fine. I will correct it.

File: .nvmrc [hunk 4]
```diff
@@ -0,0 +1 @@
+v16.13.2
```
[the remark this answers is not in the document]
Comment (vlad-dubrovski) [reply .nvmrc:1]: @iryna-markovich-vocovo https://nodejs.org/en 18.17.0

**Pull request closed without merging**